control-plane/README.md

```
@@ -0,0 +1,189 @@
+cloudflared tunnel create control-plane-tunnel
+```
+
+### B. Configure DNS
+Route a domain (e.g., `loki.yourdomain.com`) to this tunnel.
+```bash
+cloudflared tunnel route dns control-plane-tunnel loki.yourdomain.com
+```
+
+### C. Deploy Tunnel Agent
+Create a secret with your tunnel credentials JSON:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)